src/openzaak/components/documenten/openapi.yaml

```diff
@@ -7166,6 +7320,201 @@ components:
       - informatieobjecttype
       - taal
       - titel
+    EnkelvoudigInformatieObjectCreateLock:
```

Comment:
Kind of surprised to see that this is added, doesn't this same component already exist for the EnkelvoudigInformatieObjectViewSet?

Reply:
hmm okay, it's not that big of a deal I guess